pkg/gofr/datasource/solr/solr.go

```diff
@@ -193,36 +266,25 @@ func call(ctx context.Context, method, url string, params map[string]any, body i
 
 	req.URL.RawQuery = q.Encode()
 
-	client := &http.Client{}
+	req = req.WithContext(ctx)
```

Comment:
we can directly create the req with context on line 245

Reply:
sure will do